api/services/ocean/aks/paths/rightSizing.yaml

```diff
@@ -0,0 +1,23 @@
+post:
+  summary: "Fetch Right Sizing Recommendations"
+  description: >
+    Get right-sizing recommendations for an Ocean cluster and filter them according to namespace or label.
+  operationId: "oceanAzureFilterRightSizingWithFilter"
+  tags:
+    - "Ocean AZURE"
```

Comment:
Wrong Tag name: need to be 'Ocean AKS' -

Reply:
Fixed